goal-planning/goal-planning-frontend
# Feat / Maintain User Login Session

## Changes
1. Created an Express route for checking authenticated session.
2. Implemented Redux thunk logic to dispatch session checkup.
3. Move Redux store provider into the highest level hierarchy.
4. Run some Prettier formatting.

## Purpose
Keep users logged in when they refresh the page or navigate outside of the application.

## Approach
Use the Django verify token endpoint to determine session validity. Then tie this onto our Express server and bind it with our frontend redux state logic.

## Testing Steps
1. Pull down code.
2. Run application.
3. Login to the app.
4. Refresh the page and confirm you still have your logged in session running.

Closes #56
